src/style.css

```
@@ -0,0 +1,40 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url("../src/images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:first-child {
+  background-image: url("../src/images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("../src/images/star-active.svg");
+}
```

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2) {
  background-image: url("../src/images/star-active.svg");
}
```
you may list all selectors (if you add the same style for each one) to make it look more laconic

---

combine all such selectors